priv/cabbage/apps/itest/test/itest/transactions.exs

```diff
@@ -0,0 +1,174 @@
+defmodule TransactionsTests do
```

Comment:
Please add copyrights.

Reply:
we have a `credo` pass which should pick up missing licenses - it works at the root-level credo config. If the credo configs are separate for `priv` maybe let's copy it over to here?

Reply:
fixed it